### PR TITLE
FLEXID-4280 Added menu for orchestration demo applications.  Added ba…

### PR DESCRIPTION
**Pull request by AvihaiApatoff**
…ckend to support quick launch. Added onboarding simple flow sample. Added client integration simple flow sample.

### CODE REVIEW

File: orchestration/menu.html [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+                      flex: 1 1 0%;
+                      display: flex;
+                      flex-direction: row;
+                      border: 1px solid rgb(204, 204, 204);
+                      border-radius: 3px;
+                      margin: 7px;
+                      padding: 24px;
+                    "
+                  >
+                    <img
```
Comment (ts-ron) [inline orchestration/menu.html:142]: @AvihaiApatoff I believe the "consume CSM" is not a journey we can demonstrate as a standalone.
Instead - I committed a small change where the "wait for ticket" page instructs you to open a link that contains the ticket-id in another browser. 
If this makes sense to you let's remove this button

**Review by ts-ron — CHANGES_REQUESTED** [review]
So overall looks great, and good you kept the core sample that also includes some backend logic and crypto binding based on forms. 
See my comment on the menu.html - on not exposing the `consume-csm` as a separate use case

**Comment by AvihaiApatoff** [comment]
Removed the menu item for consume CSM. Some additional cosmetic fixes.

**Review by ts-ron — APPROVED** [review]
LGTM

**Comment by ShayKlig** [comment]
@AvihaiApatoff @ts-ron i think we can close this one since we have merged the changes to the main-stage branch.

**Comment by ts-ron** [comment]
> @AvihaiApatoff @ts-ron i think we can close this one since we have merged the changes to the main-stage branch.

@AvihaiApatoff do we want for now to make `main-stage` the de-facto main, until we stabilize? If so I agree w @ShayKlig

**Comment by AvihaiApatoff** [comment]
> > @AvihaiApatoff @ts-ron i think we can close this one since we have merged the changes to the main-stage branch.
> 
> @AvihaiApatoff do we want for now to make `main-stage` the de-facto main, until we stabilize? If so I agree w @ShayKlig

I dont mind removing this PR. Any reason to keep main-stage and not merge it to main?

**Comment by AvihaiApatoff** [comment]
Content has been merged into main-stage for now.

**Pull request closed without merging**